GooglePlayInstant/Editor/PlayInstantRunner.cs

```diff
@@ -31,6 +31,12 @@ public static class PlayInstantRunner
         /// </summary>
         public static void BuildAndRun()
         {
+            if(AndroidSdkManager.AndroidSdkRoot == null)
```

Comment:
Revert (done in main repo)

Reply:
Probably rebase, assuming that makes a cleaner commit history.

Ultimately you'll do a "Squash & Merge", so I'm not sure which way is better or if it matters at all.